### PR TITLE
Add IPv6 table on network page

### PR DESCRIPTION
**Pull request by sandeepasingh116**
Added IPv6 table on the network page 

- Added toggle named IPv6 Auto config
- Add ability to add IPv6 ip address
- IPv6 modal as the ability to accept IPv6 type of ip address.

### CODE REVIEW

File: src/store/modules/Settings/NetworkStore.js [hunk 1]
```diff
@@ -248,6 +334,44 @@ const NetworkStore = {
           );
         });
     },
+    async updateIpv6Address({ dispatch, state }, newIpv6Address) {
+      const originalAddresses =
+        state.networkSettings[state.selectedInterfaceIndex].staticIpv6Addresses;
+      const updatedIpv6 = originalAddresses.map((item) => {
+        const address = item.Address;
+        console.log(find(newIpv6Address, { Address: address }));
```
Comment (Nikhil-Ashoka) [inline src/store/modules/Settings/NetworkStore.js]: I think we can remove this

Comment (Nikhil-Ashoka) [reply src/store/modules/Settings/NetworkStore.js]: console.log(find(newIpv6Address, { Address: address }));

File: src/store/modules/Settings/NetworkStore.js [hunk 2]
```diff
@@ -205,6 +231,66 @@ const NetworkStore = {
           );
         });
     },
+    async saveIpv6DhcpEnabledState({ commit, state, dispatch }, dhcpState) {
+      const updatedDhcpState = dhcpState ? 'Enabled' : 'Disabled';
+      commit('setIpv6DchpEnabledState', updatedDhcpState);
```
Comment (Nikhil-Ashoka) [inline src/store/modules/Settings/NetworkStore.js:236]: Can we rename setIpv6**Dchp**EnabledState to setIpv6**Dhcp**EnabledState?

File: src/views/Settings/Network/Network.vue [hunk 3]
```diff
@@ -156,6 +182,29 @@ export default {
           .finally(() => this.endLoader());
       }
     },
+    saveIpv6Address(modalFormData) {
+      const modalData = [modalFormData];
+      console.log('ipAddress', this.ipAddress);
```
Comment (Nikhil-Ashoka) [inline src/views/Settings/Network/Network.vue]: We can remove the console log

File: src/views/Settings/Network/Network.vue [hunk 4]
```diff
@@ -156,6 +182,29 @@ export default {
...
+        const selectedRow = { Address: this.ipAddress, PrefixLength: 0 };
+        const editRow = modalData.concat(selectedRow);
+        this.$store
+          .dispatch('network/updateIpv6Address', editRow)
+          .then((message) => this.successToast(message))
+          .catch(({ message }) => this.errorToast(message))
+          .finally(() => this.endLoader());
+      } else {
+        // Add new address
+        console.log('ipAddress is empty else me aya', modalData);
```
Comment (Nikhil-Ashoka) [inline src/views/Settings/Network/Network.vue]: Same as above

File: src/store/modules/Settings/NetworkStore.js (hunk 2 above)
Comment (Nikhil-Ashoka) [reply src/store/modules/Settings/NetworkStore.js:236]: Note: this typo is in multiple locations

**Review by Nikhil-Ashoka — APPROVED** [review]

File: src/store/modules/Settings/NetworkStore.js (hunk 1 above)
Comment (sandeepasingh116) [reply src/store/modules/Settings/NetworkStore.js]: Done

File: src/views/Settings/Network/Network.vue (hunk 3 above)
Comment (sandeepasingh116) [reply src/views/Settings/Network/Network.vue]: Done

File: src/views/Settings/Network/Network.vue (hunk 4 above)
Comment (sandeepasingh116) [reply src/views/Settings/Network/Network.vue]: Done

**Pull request merged**